libnetwork/resolvconf/resolvconf_unix_test.go

```diff
@@ -407,7 +419,7 @@ func TestFilterResolvDNS(t *testing.T) {
 	}
 
 	// with IPv6 enabled, and no non-localhost servers, Google defaults (both IPv4+IPv6) should be added
-	ns0 = "\nnameserver 8.8.8.8\nnameserver 8.8.4.4\nnameserver 2001:4860:4860::8888\nnameserver 2001:4860:4860::8844"
+	ns0 = "nameserver 8.8.8.8\nnameserver 8.8.4.4\nnameserver 2001:4860:4860::8888\nnameserver 2001:4860:4860::8844\n"
```

Comment:
You could apply [`strings.TrimSpace`](https://pkg.go.dev/strings#TrimSpace) to `result.Content` to make the tests not be a change-detector for differences in leading and trailing whitespace.